Mage/src/main/java/mage/abilities/effects/common/CreateTokenTargetEffect.java

```diff
@@ -97,9 +97,9 @@ public String getText(Mode mode) {
             if (token.getDescription().endsWith("token")) {
                 sb.append("s");
             }
-            int tokenLocation = sb.indexOf("token");
+            int tokenLocation = sb.indexOf("token ");
             if (tokenLocation != -1) {
```

Comment:
Just a heads up, I made this same change in #10359 which is not yet merged. Hopefully that doesn't lead to a conflict.

Reply:
if it's the exact same change then it should be fine